devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy

```diff
@@ -132,9 +132,7 @@ private Map translateZoneStatus(ZoneStatus zs) {
     // Some sensor models that use this DTH use alarm1 and some use alarm2 to signify motion
     if (isDSM300()) {
     	return (zs.isAlarm1Set() || zs.isAlarm2Set()) ? getContactResult('open') : getContactResult('closed')
-    } else {    
-    	return (zs.isAlarm1Set() || zs.isAlarm2Set()) ? getMotionResult('active') : getMotionResult('inactive')
-    } 
```

Comment:
Was it intentional ?
If You leave it like this then this function will not return anything for models other than **isDSM300**

Reply:
@KKlimczukS Yes. Other models is unnecessary code.